trendly/src/app/histogram-section/histogram-section.component.ts

```diff
@@ -0,0 +1,161 @@
+import {Component, HostListener, Input, OnInit, SimpleChanges} from '@angular/core';
+
+import {ColorsService} from '../colors.service';
+
+interface chartRole {
+  role: string;
+}
+type ColumnArray = (chartRole|string)[];
```

Comment:
Use Array<type> for complex array types (and only in these cases)
```suggestion
type ColumnArray = Array<chartRole|string>;
```

Reply:
Done.